test/com/google/enterprise/adaptor/DocumentHandlerTest.java

```diff
@@ -698,6 +698,42 @@ public void getDocContent(Request request, Response response)
   }
 
   @Test
+  public void testForcedDecision() throws Exception {
+    List<MetadataTransform> transforms = new LinkedList<MetadataTransform>();
+    transforms.add(new MetadataTransform() {
+      @Override
+      public void transform(Metadata metadata, Map<String, String> params) {
+        params.put("Forced-Transmission-Decision", "as-is");
```

Comment:
Neither was in place at the time, but now I setForcedTransmissionDecision in the response.